packages/flutter/lib/src/widgets/form.dart

```diff
@@ -127,6 +135,18 @@ class Form extends StatefulWidget {
   /// will rebuild.
   final VoidCallback onChanged;
 
+  /// Used to enable/disable form fields auto validation and update their error text.
+  ///
+  /// If [AutoValidateMode.onUserInteraction] this form will only auto-validate
+  /// after its content changes, if [AutoValidateMode.always] it will auto
+  /// validate even without user interaction and if [AutoValidateMode.disabled]
+  /// the auto validation will be disabled.
```

Comment:
```suggestion
  /// If [AutoValidateMode.onUserInteraction], this form will only auto-validate
  /// after its content changes. If [AutoValidateMode.always], it will auto
  /// validate even without user interaction. If [AutoValidateMode.disabled],
  /// the auto validation will be disabled.
```